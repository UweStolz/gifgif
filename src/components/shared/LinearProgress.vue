<template
  functional
>
  <v-progress-linear
    indeterminate
    color="red"
  />
</template>

<script lang="ts">
import Vue from 'vue';

export default Vue.extend({
  name: 'LinearProgress',
  functional: true,
});
</script>
