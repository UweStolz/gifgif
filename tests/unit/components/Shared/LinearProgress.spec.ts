import LinearProgress from '@/components/shared/LinearProgress.vue';
import shallow from '../../../helper';

describe('LinearProgress.vue', () => {
  it('renders properly', () => {
    const wrapper = shallow(LinearProgress);
    expect(wrapper.element).toMatchSnapshot();
  });
});
