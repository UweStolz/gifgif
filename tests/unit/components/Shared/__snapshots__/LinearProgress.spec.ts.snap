// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinearProgress.vue renders properly 1`] = `
<div
  aria-valuemax="100"
  aria-valuemin="0"
  class="v-progress-linear theme--light"
  role="progressbar"
  style="height: 4px;"
>
  <div
    class="v-progress-linear__background red"
    style="opacity: 0.3; left: 0%; width: 100%;"
  />
  <div
    class="v-progress-linear__buffer"
  />
  <div
    class="v-progress-linear__indeterminate v-progress-linear__indeterminate--active"
  >
    <div
      class="v-progress-linear__indeterminate long red"
    />
    <div
      class="v-progress-linear__indeterminate short red"
    />
  </div>
</div>
`;
